packages/endomoat/src/policy-converter.js

```diff
@@ -16,7 +16,7 @@ import { readPolicy } from './policy.js'
 const { isArray } = Array
 const { entries, fromEntries } = Object
 
-const DEFAULT_ATTENUATOR = '@lavamoat/endomoat/attenuator'
+export const DEFAULT_ATTENUATOR = '@attennuator@' // If we wanted endo to load and execute the code in attenuators compartment, we could pass '@lavamoat/endomoat/attenuator' as default attenuator and have it loaded statelessly. We're using an impossible specifier to match with an external module instead.
```

Comment:
In other words, we use the impossible specifier to trigger the exit module hook?